docs/ref/settings.txt

```diff
@@ -1850,18 +1850,28 @@ Default::
 
     {}  # empty dictionary
 
-A dictionary specifying the package where migration modules can be found on a per-app basis. The default value
-of this setting is an empty dictionary, but the default package name for migration modules is ``migrations``.
+A dictionary specifying the package where migration modules can be found on a
+per-app basis. The default value of this setting is an empty dictionary, but
+the default package name for migration modules is ``migrations``.
 
 Example::
 
     {'blog': 'blog.db_migrations'}
 
-In this case, migrations pertaining to the ``blog`` app will be contained in the ``blog.db_migrations`` package.
+In this case, migrations pertaining to the ``blog`` app will be contained in
+the ``blog.db_migrations`` package.
 
 If you provide the ``app_label`` argument, :djadmin:`makemigrations` will
 automatically create the package if it doesn't already exist.
 
+.. versionchanged:: 1.9
+
+When you supply ``None`` as a value for an app, Django will consider the app as
+an app without migrations regardless of an existing ``migrations`` submodule.
+This can for example be used to skip migrations for this app in tests. If this
```

Comment:
This can be used, for example, in a test settings file to skip migrations while testing (tables will still be created for the apps' models).